nextcloudappstore/core/static/assets/js/ui/app-edit.js

```diff
@@ -0,0 +1,19 @@
+(function (global) {
+    'use strict';
+
+    // App ownership transfer form
+    let ownerTransferForm = document.getElementById('app-ownership-transfer-form');
+    let appId = ownerTransferForm.getAttribute('data-app-id');
+    let promptMsg = ownerTransferForm.getAttribute('data-prompt');
+    let failMsg = ownerTransferForm.getAttribute('data-fail');
+    ownerTransferForm.addEventListener('submit', (event) => {
+        event.preventDefault();
+        let appIdInput = window.prompt(promptMsg);
```

Comment:
Should be implemented similarly to the delete app form in the user profile. It's almost never a good idea to use alert or prompt